File: scratch/fix_links.js
const fs = require('fs');
let c = fs.readFileSync('index.html', 'utf8');
c = c.split('href="./docs/').join('href="https://ifms.punjab.gov.in/docs/');
c = c.split('bi-caret-right-fill').join('bi-download');
fs.writeFileSync('index.html', c);

File: scratch/fix_links.py
import re

with open('index.html', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace local links with absolute live URLs
content = content.replace('href="./docs/', 'href="https://ifms.punjab.gov.in/docs/')

# Replace the caret icons with download icons
content = content.replace('bi-caret-right-fill', 'bi-download')

with open('index.html', 'w', encoding='utf-8') as f:
    f.write(content)
